Include/internal/pycore_pymem.h

```diff
@@ -100,6 +100,18 @@ static inline void _PyObject_XDecRefDelayed(PyObject *obj)
 }
 #endif
 
+#ifdef Py_GIL_DISABLED
+// This is the delayed-free equivalent of Py_XSETREF(), providing the same
```

Comment:
this sounds opposite, `_PyObject_XSetRefDelayed` is the delayed version whereas `Py_XSETREF` is the delayed free equivalent